data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="E_MOVE" Comment="Data latch (d) flip flop for ANY Datatype">
+	<Identification Standard="61499" Classification="Sample and Hold" Description="Copyright (c) 2013,2024 TU Wien ACIN,  HR Agrartechnik GmbH&#10; &#10;This program and the accompanying materials are made&#10;available under the terms of the Eclipse Public License 2.0&#10;which is available at https://www.eclipse.org/legal/epl-2.0/&#10;&#10;SPDX-License-Identifier: EPL-2.0&#10;&#10;This Block is functional identic to E_D_FF but for ANY Datatype. &#10;it reduces the Number of Events on the Output CNF.&#10;&#10;This Block take the input IN compare it with the output OUT and only if it is not equal forward the event REQ to CNF.&#10;&#10;for REAL and LREAL the Function of this Block is not guaranteed, because of the special cases of rounding Errors in REAL and LREAL." >
+	</Identification>
+	<VersionInfo Organization="TU Wien ACIN" Version="1.0" Author="Monika Wenger" Date="2013-08-17">
+	</VersionInfo>
+	<VersionInfo Organization=" HR Agrartechnik GmbH" Version="1.1" Author="Franz Höpfinger" Date="2024-10-31" Remarks="Copy F_MOVE to E_MOVE and adjust Comments">
+	</VersionInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="REQ" Type="Event" Comment="Normal Execution Request">
+				<With Var="IN"/>
+			</Event>
+		</EventInputs>
+		<EventOutputs>
+			<Event Name="CNF" Type="Event" Comment="Execution Confirmation">
+				<With Var="OUT"/>
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN" Type="ANY" Comment="input"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="OUT" Type="ANY" Comment="input = output"/>
+		</OutputVars>
+	</InterfaceList>
+	<Attribute Name="GenericClassName" Value="'GEN_FORTE_E_MOVE'"/>
```

Comment:
This attribute should only be set if the type is on FORTE a generic one with a flexible number of in and outputs. As this is not the case please remove it.

Reply:
Ah. Ok. But for F_MOVE this is the case?

Reply:
https://github.com/eclipse-4diac/4diac-ide/pull/688

Reply:
Yes F_MOVE needs it!!!

Reply:
Interesting to know.